addons/survey/survey.py

```diff
@@ -200,7 +205,9 @@ def _get_result_url(self, cr, uid, ids, name, arg, context=None):
             'Email Template', ondelete='set null'),
         'thank_you_message': fields.html('Thank you message', translate=True,
             help="This message will be displayed when survey is completed"),
-        'quizz_mode': fields.boolean(string='Quizz mode')
+        'quizz_mode': fields.boolean(string='Quizz mode'),
+        'is_answer': fields.function(_is_answer,
+            string="Is Answer", type="boolean"),
```

Comment:
What is the purpose of this field? A user_input is a container for all the answers of one particular user